### DISCUSSION TITLE
Find sosetup file

### DISCUSSION THREAD

**Discussion by dwlr09** [Security-Onion-Solutions/securityonion#12261] [Unsupported Versions]
Made a typo in on my gateway. Wondering how to get to the sosetup file on my heavy node so I can just change the gateway without having to go through the rebuild process. I can't seem to find it in the root anymore just the .log filw.

**Comment by cm-ops** [comment]
Did you try changing it with `nmcli`? `nmcli con mod $INT ipv4.gateway "XX.XX.XX.XX"` Then restart NetworkManager.
